CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# v2.4.1 (unreleased)
+# v2.5.0 (unreleased
+- This release supports OneView Rest API versions 800/1000/1200 minimally for Server Certificate.
```

Comment:
You dont need to specify the resource here also just say for the resources listed below.